src/com/gh4a/utils/ApiHelpers.java

```diff
@@ -84,4 +87,25 @@ public static boolean loginEquals(User user, String login) {
     public static boolean loginEquals(String user, String login) {
         return user != null && user.equalsIgnoreCase(login);
     }
+
+    public static List<User> getUniqueUsers(List<User> users) {
+        List<User> uniqueUsers = new ArrayList<>();
+
+        for (User user : users) {
+            boolean isUnique = true;
+            for (int i = 0; i < uniqueUsers.size(); i++) {
+                final User existingUser = uniqueUsers.get(i);
+
+                if (existingUser.getId() == user.getId()) {
+                    isUnique = false;
+                    break;
+                }
+            }
+            if (isUnique) {
+                uniqueUsers.add(user);
+            }
+        }
+
+        return uniqueUsers;
```

Comment:
That's indeed a better option. I would recommend doing this for all the API models that have some sort of ID. It will make it easier to work with them in future.